### DISCUSSION TITLE
Suggested change is having problem with missing node extension

### DISCUSSION THREAD

**Discussion by danieltoomey** [renovatebot/renovate#23545] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

The actual coding cannot be released.  I think the description below demarks the issue.

Build is hitting the issue `'computedNodeDir' for extension 'node' of type com.github.gradle.node.NodeExtension`.

The line of the build script with issue is: `println nodeSetup.nodeDir.get()`

Tried solution seen on internet `println node.computedNodeDir.get()`, but that does not work either.

Unclear on why Renovate recommending change that does not build.  I am guessing a node extension should be installed that would present the nodeDir item for use?

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Renovate cannot guarantee that each update it proposes will build. That's up to you and your tests to verify. There's nothing wrong or to be changed here

**Comment by danieltoomey** [comment]
Ok.  I did not understand that.  Is there a way to mark the items in the GIT ticket so the system moves on?

**Reply by rarkins** [reply]
You can choose to either:
- ignore this particular update by closing the PR
- ignore the dependency forever by adding it to ignoreDeps
- fix the PR yourself by adding to the branch
